pkg/apis/ceph.rook.io/v1/network.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2019 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+func IsHostNetworkDefined(net NetworkSpec) bool {
```

Comment:
Could you define this method as: 
```
func (net *NetworkSpec) IsHostNetwork() bool {
  return (net.HostNetwork && net.Provider == "") || net.Provider == "host"
}
```

Then it is cleaner for the usage such as:
```
net.HostNetwork()
```

Reply:
Did you mean to go with `net.IsHostNetwork()`? But while we are at it, I think its better to shorten it further just to `net.IsHost()`, no need to repeat the network context again.